ion-schema/src/types.rs

```diff
@@ -445,12 +490,26 @@ impl TypeValidator for TypeDefinitionImpl {
         Err(Violation::with_violations(
             type_name,
             ViolationCode::TypeConstraintsUnsatisfied,
-            "value didn't satisfy type constraint(s)",
+            &"value didn't satisfy type constraint(s)".to_owned(),
```

Comment:
Why does this need to change? It doesn't look like you changed the function signature.

Reply:
Changed this. I don't know how it was working before though.